packages/datetime/src/_datepicker.scss

```diff
@@ -109,6 +109,18 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
     display: table-row;
   }
 
+  .DayPicker-WeekNumber {
+    display: table-cell;
+    cursor: pointer;
```

Comment:
hmm wondering about this line...

clicking on a week number produces a console error inside ReactDayPicker:

```
Uncaught TypeError: onWeekClick is not a function
    at onClick (Month.js:70)
    at Object.invokeGuardedCallback (ReactErrorUtils.js:26)
    at executeDispatch (EventPluginUtils.js:85)
    at Object.executeDispatchesInOrder (EventPluginUtils.js:108)
    at executeDispatchesAndRelease (EventPluginHub.js:43)
    at executeDispatchesAndReleaseTopLevel (EventPluginHub.js:54)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (forEachAccumulated.js:24)
    at Object.processEventQueue (EventPluginHub.js:254)
    at runEventQueueInBatch (ReactEventEmitterMixin.js:17)
```

Reply:
something like that. might also be a bug in react-day-picker cuz it should handle optional callbacks? worth checking their release notes to see if we should update our dep.

Reply:
`pointer-events: none` could do the trick too?

In any case, let's remove this and have a default cursor. No need for a pointer if it looks disabled and does nothing on click

Reply:
but onWeekClick can be added as a dayPickerProps so it can actually do something

Reply:
right, but it doesn't do anything by default and looks disabled, which isn't good for interaction. is it possible to tweak the styling when `onWeekClick` is defined? not even sure that's a great idea, but we do need proper styling